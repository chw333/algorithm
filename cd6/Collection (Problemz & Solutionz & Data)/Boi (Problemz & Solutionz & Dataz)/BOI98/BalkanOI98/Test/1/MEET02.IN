135 9 119
7 1 1
7 3 5
12 11 2
17 5 2
19 12 3
20 8 6
22 9 1
22 15 7
26 1 1
26 7 1
26 23 3
27 2 2
30 20 5
32 5 8
34 20 9
35 3 3
38 10 7
39 2 8
42 40 1
44 6 4
46 18 7
47 25 8
49 10 9
49 44 8
50 8 6
51 2 9
51 11 5
52 43 7
53 4 5
54 7 5
57 31 8
58 1 1
58 7 2
58 26 2
58 54 3
59 48 1
60 53 5
65 9 6
65 14 9
66 57 9
67 1 1
67 44 1
68 62 7
69 53 5
70 16 8
71 36 5
72 28 9
72 33 3
72 58 2
74 9 5
74 71 10
75 54 3
76 18 8
77 73 7
78 77 5
79 71 2
80 59 8
80 63 6
81 56 5
82 74 2
83 14 2
84 55 10
85 82 7
86 41 1
87 10 7
87 84 10
88 21 7
88 60 6
89 12 8
90 45 3
90 65 5
91 18 4
91 37 4
91 61 5
91 68 3
92 26 3
93 23 8
93 24 8
94 43 4
95 13 2
96 32 4
96 53 4
98 57 6
99 50 4
100 9 1
100 47 10
101 56 5
101 67 3
102 15 8
103 65 2
105 45 2
106 62 3
106 64 8
107 34 6
108 38 9
110 36 4
111 47 2
111 99 9
112 4 6
112 29 9
112 40 10
113 75 9
114 27 8
114 70 5
114 104 3
114 113 2
115 90 6
116 9 2
116 40 7
116 92 9
116 108 7
117 35 6
117 41 6
117 64 5
118 59 3
118 93 3
119 17 3
119 31 4
119 65 7
119 116 8
120 28 8
120 94 3
120 109 6
121 73 9
122 116 3
123 33 3
123 95 8
123 102 10
124 10 7
125 66 10
126 121 1
127 83 4
127 97 5
127 122 7
127 126 8
