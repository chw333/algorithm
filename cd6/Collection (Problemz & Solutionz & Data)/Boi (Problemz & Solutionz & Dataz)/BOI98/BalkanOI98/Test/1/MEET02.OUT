9 74 9
119 116 9
